contract/scripts/wait-for-container-ready.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I would recommend renaming this script to `wait-for-blocks`

Reply:
fixed.